src/opencl_bitlocker_fmt_plug.c

```diff
@@ -358,9 +353,9 @@ static void done(void)
 
 static void *get_salt(char *ciphertext)
 {
-	static char dummy[BITLOCKER_JTR_HASH_SIZE_CHAR + 1];
+	static char dummy[PLAINTEXT_LENGTH + 1];
 
-	return strnzcpy(dummy, ciphertext, BITLOCKER_JTR_HASH_SIZE_CHAR + 1);
+	return strnzcpy(dummy, ciphertext, PLAINTEXT_LENGTH + 1);
```

Comment:
This doesn't look right. PLAINTEXT_LENGTH is the max length of a password *candidate*. The figure needed here is the maximum length ever needed to represent a "hash" including the ending NULL byte.